src/Nox.Types/Types/Color/Color.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Color"/> type and value object. 
+/// </summary>
+public class Color : ValueObject<(byte Alpha, byte Red, byte Green, byte Blue), Color>
```

Comment:
Have you considered storing color as a single value?
I don't think there's any having them like this if we won't use them separately.
It also adds to complexity when storing/retrieving these values that could be represented by a single number/string.

Reply:
The requirement was to persist it as 4 bytes.
Maybe "byte[]", but you have to check how it would be persisted in the database.

![image](https://github.com/NoxOrg/Nox.Generator/assets/131167535/43940033-3c4f-4c0b-b55d-7c8ba743dc09)

Reply:
Each element of the tuple should correspond to a column in a table. For this tuple representing RGBA color values, we can create a table with four columns (Red, Green, Blue, Alpha). and then store the tuple's data. Similarly, to retrieve the tuple, execute a SELECT and construct the tuple back from the query result.